File: natural.c
//This program prints first n natural numbers.

#include<stdio.h>

int main(){
    int n;
    printf("Enter the Range :\n");
    scanf("%d",&n);
    printf("---Printing first n natural numbers---\n");

    for(int i=1;i<=n;i++){
        printf("%d\n",i);
    } 

    return 0;
}

File: Java/natural.java
//This program prints first n natural numbers.

package Java_Samples;
import java.util.Scanner;

public class natural {
    public static void main(String [] args){
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the Range : ");
        int n = sc.nextInt();

        System.out.println("---Printing first n natural numbers---");
        for(int i=1;i<=n;i++){
            System.out.println(i);
        }

        sc.close();
    }
    
}
